### ISSUE TITLE
upgrade-handle should report correct path in its ERROR msg

### ISSUE THREAD

**Issue by yarikoptic** [datalad/datalad#224]
even though those aren't yet really user-oriented, '..' is really not informative
```
/tmp# datalad upgrade-handle 
2015-09-21 14:10:56,493 [ERROR  ] No handle repository found in .. (RuntimeError) (main.py:206)

```

**Comment by hanke** [comment]
And once fixed, please make them go away entirely #222 ;-)

**Comment by yarikoptic** [comment]
well -- this  one is a legitimate error message so it should be delivered one way or another (commented in 222)
